staging/src/github.com/openshift/oc/pkg/helpers/dockerclient/client.go

```diff
@@ -0,0 +1,955 @@
+package dockerv1client
```

Comment:
this is a copy?  it all looks external, why not a candidate for library-go?

Reply:
it is mostly copy bu without the docker types and crazy conversions... the auth and interface is all the same. i will try to make the changes more obvious in commit